_docs-sources/foundations/landing-zone/enable-control-tower.md

```diff
@@ -84,38 +114,50 @@ Control Tower Creation will take around an hour to complete
 
 1. Setting up the landing zone can take up to one hour. You will see a notification like the one below with the estimated time it will take for all the resources to be created.
 
-  ![Landing Zone Setup Status](/img/devops-foundations/account/control-tower-setup-status.png)
+   ![Landing Zone Setup Status](/img/devops-foundations/account/control-tower-setup-status.png)
 
-  :::tip
+   :::tip
 
-  You can safely close the browser once you see this.
+   You can safely close the browser once you see this.
```

Comment:
I don't understand this — which browser? My initial read was the one I'm reading the docs in. But maybe you mean the AWS Control Tower console? But then the very next step instructs the user to navigate to the AWS Control Tower dashboard.

Reply:
I didn't write the original line, but I think the intent here was following up with the previous line that it can take up to an hour, and just advising that you don't need to keep the console page open for an hour while you wait (and if you accidentally do close the tab it is not harmful/won't cancel/abort the operation AWS is performing in the background).

Reply:
Ah, that makes sense. I think that context would be very useful to add to the callout. Here's an attempt:
> You can safely close your browser tab once you see this notice. The setup process will proceed unaffected in the background.